assets/state-vgpu-device-manager/0210_clusterrole.yaml

```diff
@@ -14,8 +14,6 @@ rules:
   - ""
   resources:
   - nodes
-  - pods
-  - pods/eviction
```

Comment:
I believe access to `pods/eviction` was never needed in the first place. We only watch for pods (our operands) to be deleted before applying a vGPU device configuration.